.changes/v2.19.0/537-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Remove URL checks from `CreateCatalogFromSubscriptionAsync` to allow catalog creation from non-VCD entities [GH-537]
```

Comment:
Would be useful to mention vSphere explicitly here.

Reply:
done